agent/conf/agent.properties

```diff
@@ -290,4 +290,4 @@ iscsi.session.cleanup.enabled=false
 # host.cpu.manual.speed.mhz=0
 
 # Enable/disable IO driver for Qemu (in case it is not set CloudStack can also detect if its supported by qemu)
-# enable.io.uring=true
+# enable.io.uring=true
```

Comment:
add a line ending? (maybe)

Reply:
I have added a new line to the `agent.properties` file.

Reply:
ok, giving it another try